### PR TITLE
Kafka/Flink: Improve documentation

### PR DESCRIPTION
**Pull request by amotl**
Just a small update to some bits of the corresponding documentation about the Kafka/Flink/CrateDB pipeline scenario.

### CODE REVIEW

File: stacks/kafka-flink/README.Unix.rst [hunk 1]
```diff
@@ -13,6 +14,22 @@ This document will outline how to run the tutorial on Linux, macOS and WSL2.
...
+
+- bash
+- cat
+- curl
+- docker
+- git
+- head
+- jq
+- tar
+- wget
```
Comment (matriv) [inline stacks/kafka-flink/README.Unix.rst:31]: not important, but we could replace wget with curl where needed to download a file

File: stacks/kafka-flink/README.Unix.rst [hunk 2]
```diff
@@ -13,6 +14,22 @@ This document will outline how to run the tutorial on Linux, macOS and WSL2.
...
+those programs are installed/available on your machine / in your environment.
+
+- bash
+- cat
+- curl
+- docker
+- git
+- head
+- jq
+- tar
```
Comment (matriv) [inline stacks/kafka-flink/README.Unix.rst:30]: Maybe it's important to mention that is gnu tar, not bsd tar (default tar in MacOS)?

File: stacks/kafka-flink/README.Windows.rst [hunk 3]
```diff
@@ -16,6 +16,15 @@ PowerShell. On Windows 10, please use those commands with PowerShell 3+
...
 *****
 
+Prerequisites
+=============
+
+In order to exercise the command walkthrough successfully, please make sure
+those programs are installed/available on your machine / in your environment.
+
+- docker
+- tar
```
Comment (matriv) [inline stacks/kafka-flink/README.Windows.rst:26]: same here.

File: stacks/kafka-flink/README.rst [hunk 4]
```diff
@@ -36,26 +36,25 @@ About
...
 It will outline how to acquire and publish data to Kafka, process it using
-Flink and store the data stream into CrateDB.
+Flink, and store the data stream into CrateDB.
 It describes this system as a part of the CrateDB reference architecture and
 can be used as a blueprint for building own applications.
 
 The following versions of software components are used:
 
-- Confluent Kafka 6.1.1
 - Apache Flink 1.12
```
Comment (matriv) [inline stacks/kafka-flink/README.rst:45]: Just a reminder to sync with: https://github.com/crate/cratedb-examples/pull/23 regarding the updated versions.

File: stacks/kafka-flink/backlog.rst [hunk 5]
```diff
@@ -9,22 +9,45 @@ Functionality
 
 Data type mapping
 =================
+- [o] Currently, the DDL schema is ``CREATE TABLE "taxi_rides" ("payload" OBJECT(DYNAMIC))``.
+  Shell there be an advanced version, where data is stored into individual columns?
```
Comment (matriv) [inline stacks/kafka-flink/backlog.rst:13]:
```suggestion
  Shall there be an advanced version, where data is stored into individual columns?
```

File: stacks/kafka-flink/README.Unix.rst (hunk 2 above)
Comment (amotl) [reply stacks/kafka-flink/README.Unix.rst:30]: Maybe BSD tar works in this case equally well?

File: stacks/kafka-flink/README.rst (hunk 4 above)
Comment (amotl) [reply stacks/kafka-flink/README.rst:45]: Yeah. Bumped to 6.1.9 there - thanks!

File: stacks/kafka-flink/README.Unix.rst (hunk 1 above)
Comment (amotl) [reply stacks/kafka-flink/README.Unix.rst:31]: Tha's true. I will consider it for a future iteration.

File: stacks/kafka-flink/README.Unix.rst (hunk 2 above)
Comment (matriv) [reply stacks/kafka-flink/README.Unix.rst:30]: if a tarball is created with gnu tar then the bsd tar most probably won't be able to extract it.

Comment (amotl) [reply stacks/kafka-flink/README.Unix.rst:30]: It works.

```
tar -xzf nyc-yellow-taxi-2017.tar.gz
```
```
tar --version
bsdtar 3.3.2 - libarchive 3.3.2 zlib/1.2.11 liblzma/5.0.5 bz2lib/1.0.6
```

Comment (matriv) [reply stacks/kafka-flink/README.Unix.rst:30]: ok, cool, thx for checking, I have encountered problems with gnu vs bsd in the past, https://xorl.wordpress.com/2012/05/15/admin-mistakes-gnu-bsd-tar-and-posix-compatibility/

**Pull request merged**